### ISSUE TITLE
Add support for related items

### ISSUE THREAD

**Issue by navilan** [hyde/hyde-old#3]
Essentially this would mean adding support for Categories/Tags. Can be integrated as a part of hyde by directly supporting Categories property on `Page` and a method for retrieving related pages in `SiteNode` or as a content processor.

**Comment by jacqueminv** [comment]
For this one we may have several strategies:
- simply take some older posts in the same category as related item
- perform a Latent Semantic Analysis (that sounds smart :)

I need time to prototype the second one, but we could implement the 1st easily and quickly I guess. The 2nd solution may take much more time to generate a site but this has to be prototyped.

Resource for LSA from the google group: http://blog.josephwilk.net/projects/latent-semantic-analysis-in-python.html

**Comment by navilan** [comment]
Yep. We can actually integrate option #1 with the CategoryManager .
